content/announcing-official-typescript-types-public-preview.md

```diff
@@ -0,0 +1,417 @@
+---
+title: Announcing the Official TypeScript Types Public Preview
+authors:
+  - chris-krycho
+date: 2022-09-16T22:00:00.000Z
+tags:
+  - '2022'
+  - typescript
+
+---
+
+As of `ember-source@4.8.0-beta.2`, Ember is shipping a public preview of our official TypeScript support for the framework itself. This is the next step in implementing [RFC 0724: Official TypeScript Support][0724] and [RFC 0800: TypeScript Adoption Plan][0800]. Anyone using TypeScript with Ember 4.8.0 Beta 2 or later can opt into using these preview types by removing the corresponding `@types` packages and adding the following import in your `types/<your app>/index.d.ts` file:
+
+```ts
+import 'ember-source/types';
+import 'ember-source/types/preview';
+```
+
+This will set your app up to start using Ember's preview types *now* and to automatically benefit as we stabilize our types incrementally over the releases ahead. You won't have to do anything except add these once and then upgrade your app on your normal upgrade cadence!
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/lHQB7DxgFbg" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+[0724]: https://rfcs.emberjs.com/id/0724-road-to-typescript/
+[0800]: https://rfcs.emberjs.com/id/0800-ts-adoption-plan/
+
+Note that there are some significant changes to these types compared to the types as they exist on DefinitelyTyped today. **All public API remains supported, but in line with RFC 0800, we intentionally provide only minimal support for Ember Classic APIs around class definitions.** Accordingly, you should migrate to native classes *before* attempting to adopt these types if you have not already done so!
+
+The rest of this post is broken into four sections:
+
+- [How the preview period will work](#toc_how-the-preview-period-will-work)
+- [Migrating from DefinitelyTyped](#toc_migrating-from-definitelytyped)
+- [New TypeScript users](#toc_new-typescript-users)
+- [What's next on Ember’s road to TypeScript?](#toc_whats-next-on-embers-road-to-typescript)
```

Comment:
On clicking this hyperlink, it does not jump to `What's next on Ember’s road to TypeScript?` section. I think due to the apostrophe present in  word `Ember's`